lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -1753,6 +1753,23 @@ defmodule Kernel.SpecialForms do
   catch values also support `:error`, as in Erlang, although it is
   commonly avoided in favor of raise/rescue control mechanisms.
 
+  ## After clauses
+
+  An `after` clause allows you to define cleanup logic that will be invoked both
+  when the tried block of code succeeds and also when an error is raised. Note,
+  however, that if a linked process exits in the `try` block, the process will
```

Comment:
It may not be a linked process rather an exit signal, which is perhaps due to a linked process exiting. Perhaps "Note that the process will exit as usually when receiving an exit signal that causes it to exit abruptly and so the `after` clause is not guaranteed to be executed".